docs/userman/gui/extension/accessing_assets.md

```diff
@@ -0,0 +1,187 @@
+- The `logo_with_text` element includes two properties: `text` and `logo_path`.
+- The `text` property has the type `PropertyType.string^`, meaning it holds a string value.
+- The `logo_path` property has the type `PropertyType.string^`, meaning it holds a string value.
+  The `default_value` parameter is set to the base64-encoded image file, which is used as the default value for the property.
+
+## Creating the React component {data-source="gui:doc/extension/example_library/front-end/src/LogoWithText.tsx"}
+```tsx title="LogoWithText.tsx"
+import React from "react";
+import { useDynamicProperty } from "taipy-gui";
+
+interface CaptionProps {
+    text: string;
+    defaultText: string;
+    logoPath: string;
+}
+
+const styles = {
+    container: {
+        display: "flex",
+        alignItems: "center",
+    },
+    logo: {
+        width: "4em",
+        height: "4em",
+        marginRight: "10px",
+    },
+};
+
+const LogoWithText = ({ text, defaultText, logoPath }: CaptionProps) => {
+    const value = useDynamicProperty(text, defaultText, "");
+
+    return (
+        <div style={styles.container}>
+            <img
+                src={`data:image/png;base64,${logoPath}`}
+                alt="LogoWithText"
+                style={styles.logo}
+            />
+            <div>{value}</div>
+        </div>
+    );
+};
+
+export default LogoWithText;
+```
+
+## Exporting the React component {data-source="gui:doc/extension/example_library/front-end/src/index.ts"}
+
+When the component is entirely defined, it must be exported by the library's JavaScript bundle.
+This is done by adding the export directive in the file *<project dir>/<package dir>front-end/src/index.ts*.
+
+```ts title="index.ts"
+import LogoWithText from "./LogoWithText";
+
+export { LogoWithText };
+```
+
+## Using the element {data-source="gui:doc/extension/logo_with_text.py"}
+
+```python title="logo_with_text.py"
+name = "Taipy"
+
+page = """
+<|{name}|logo_with_text|>
+"""
+```
+
+When you run this application, the page displays the element like this:
+<figure>
+    <img src="../logo_with_text-d.png" class="visible-dark"/>
+    <img src="../logo_with_text-l.png" class="visible-light"/>
+    <figcaption>Logo with text</figcaption>
+</figure>
+
+# Additional resources {data-source="gui:doc/extension/example_library/example_library.py#L92"}
+
+Beside the `ElementLibrary.get_resource()^` method, Taipy provides other methods to manage resources securely.
+One of them is the `ElementLibrary.get_scripts()^` method, which allows you to include JavaScript files in your application.
+This method ensures that only authorized scripts are loaded, protecting your application from potential security threats.
+
+The code snippet below illustrates how to use the `(ElementLibrary.)get_scripts()^` method to include a JavaScript file that adds animation to your application’s logo.
```

Comment:
I would swap the description of the JS script with the part where you explain how to reference it in get_scripts().